hedera-node/hedera-app/build.gradle.kts

```diff
@@ -86,7 +86,7 @@ dependencies {
         xtestImplementation(gav("com.github.spotbugs.annotations"))
         xtestImplementation(gav("com.hedera.pbj.runtime"))
         xtestImplementation(gav("com.swirlds.common"))
-        xtestImplementation(gav("com.swirlds.config"))
+        xtestImplementation(gav("com.swirlds.config.api"))
```

Comment:
is `xtest` new? I know `test`, `itest` and `e2etest` but never seen `xtest`. What is it doing?

Reply:
It is not new. It is only used here in `hedera-node/hedera-app` as far as I can tell. In the comment where it is set up it says:

```
// Add the X-test task for executing "cross-service" tests
```

Reply:
xtest is new-ish, Services is adding tests that are not integration tests or end-to-end tests, but also are not purely unit tests (basically service or module level tests).  Those are being added under `xtest` to distinguish from the other alternatives.  Ideally `xtest` will still use relatively simple testing structures and run quickly without the heavy requirements of `itest` or `e2etest`.